src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -0,0 +1,245 @@
+		set => SetValue(CommandParameterProperty, value);
+	}
+
+	/// <summary>
+	/// Command to execute when <see cref="IsExpanded"/> changed.
+	/// </summary>
+	public ICommand? Command
+	{
+		get => (ICommand?)GetValue(CommandProperty);
+		set => SetValue(CommandProperty, value);
+	}
+
+	Grid ContentGrid => (Grid)base.Content;
+
+	static void OnContentPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+	{
+		var expander = (Expander)bindable;
+		if (newValue is View view)
+		{
+			view.SetBinding(IsVisibleProperty, new Binding(nameof(Expander.IsExpanded), source: bindable));
+
+			expander.ContentGrid.Remove(oldValue);
+			expander.ContentGrid.Add(newValue);
+			expander.ContentGrid.SetRow(view, expander.Direction switch
+			{
+				ExpandDirection.Down => 1,
+				ExpandDirection.Up => 0,
+				_ => throw new NotSupportedException($"{nameof(ExpandDirection)} {expander.Direction} is not yet supported")
+			});
+
+			var temp = view.IsVisible;
+		}
+	}
+
+	static void OnHeaderPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+	{
+		var expander = (Expander)bindable;
+		if (newValue is View view)
+		{
+			expander.SetHeaderGestures(view);
+
+			expander.ContentGrid.Remove(oldValue);
+			expander.ContentGrid.Add(newValue);
+
+			expander.ContentGrid.SetRow(view, expander.Direction switch
+			{
+				ExpandDirection.Down => 0,
+				ExpandDirection.Up => 1,
+				_ => throw new NotSupportedException($"{nameof(ExpandDirection)} {expander.Direction} is not yet supported")
+			});
+		}
+	}
+
+	static void OnIsExpandedPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+	{
+		var expander = (Expander)bindable;
+		ForceUpdateLayoutSizeForItemsView(expander);
+
+		((IExpander)bindable).ExpandedChanged(((IExpander)bindable).IsExpanded);
+	}
+
+	static void OnDirectionPropertyChanged(BindableObject bindable, object oldValue, object newValue) =>
+		((Expander)bindable).HandleDirectionChanged((ExpandDirection)newValue);
+
+	void HandleDirectionChanged(ExpandDirection expandDirection)
+	{
+		switch (expandDirection)
+		{
+			case ExpandDirection.Down:
+				ContentGrid.SetRow(Header, 0);
+				ContentGrid.SetRow(Content, 1);
+				break;
+
+			case ExpandDirection.Up:
+				ContentGrid.SetRow(Header, 1);
+				ContentGrid.SetRow(Content, 0);
+				break;
+
+			default:
+				throw new NotSupportedException($"{nameof(ExpandDirection)} {expandDirection} is not yet supported");
+		}
+	}
+
+	void SetHeaderGestures(in IView header)
+	{
+		var headerView = (View)header;
+		headerView.GestureRecognizers.Remove(tapGestureRecognizer);
+		headerView.GestureRecognizers.Add(tapGestureRecognizer);
+	}
+
+	static void ForceUpdateLayoutSizeForItemsView(in Element parent)
+	{
+		var element = parent;
+
+		while (element is not null)
+		{
+			if (element is ListView listView)
+			{
+				foreach (var child in listView.AllChildren.OfType<Cell>())
+				{
+					child.ForceUpdateSize();
+				}
```

Comment:
Here's my recommendation to remove `ListView` support from this release of `Expander`

```suggestion
			if (element is ListView listView)
			{
				throw new NotSupportedException($"{nameof(Expander)} is not yet supported on `ListView`");
```

Reply:
I suggest keeping it, because ListView works much better on WinUI.